client/src/test/java/io/pravega/client/stream/impl/ControllerImplTest.java

```diff
@@ -1239,155 +1247,4 @@ public void testParallelGetCurrentSegments() throws Exception {
         ExecutorServiceHelpers.shutdown(executorService);
         assertTrue(success.get());
     }
```

Comment:
Where did these tests go?

Reply:
These tests and scnearios are now covered on the controller side as the assumption of continuous segment numbering no longer holds true. So this processing and business logic moves to controller.
There are two places where we cover these tests
1. ControllerServiceWithZKStreamTest.java
2. TableHelperTest.java

In tableHelperTest we have taken the exact same segment-time-series as used in the tests here. And it includes more cases and ControllerServiceWithZKStreamTest covers some additional complex cases.